cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
+
+    func callbacks() {
+        view.backgroundColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 1)
+        customView.didTapButton = {
+            if let payment = self.payment {
+                self.delegate.didFinishFlow(amount: payment.amount
+                                            currency: payment.currency)
+            }
+        }
+    }
+
+    func viewDidLoad() {
+        navigationController?.setNavigationBarHidden(false, animated: false)
+        customView.centerXAnchor.constraint(equalTo: view.centerXAnchor)
+        customView.centerYAnchor.constraint(equalTo: view.centerYAnchor)
+        view.addSubview(customView)
+        fetchPayment()
+        setupCallbacks()
+    }
```

Comment:
- You are missing **_override_** keyword here 
- this time it is unnecessary but always considers calling **_super.viewDidLoad_** when overriding
-  you can't set anchors before adding the view to view hierarchy, possible solutions:
      1. create new function **_func constraintUI()_** and move anchoring there, then call this func after setup all views in view hierarchy
      2. bonus one -> use defer :D to move this to the end of the scope
 - I assume that PaymentView has translateAutoresizing mask set into false, and has constrained width and height or is automatically resizing 
```suggestion
   override func viewDidLoad() {
       super.viewDidLoad()
        navigationController?.setNavigationBarHidden(false, animated: false)
        setupUI()
        constraintUI()
        fetchPayment()
        setupCallbacks()
    }
     private func setupUI() {
        self.view.addSubview(self.customView)
    }
    
    private func constraintUI() {
        customView.centerXAnchor.constraint(equalTo: view.centerXAnchor)
        customView.centerYAnchor.constraint(equalTo: view.centerYAnchor)
    }
```